testsuite/tests/typing-short-paths/errors.ml

```diff
@@ -34,3 +34,35 @@ Line 7, characters 9-18:
 Error: This expression has type "c"
        It has no method "bar"
 |}]
+
+type _ ty = Char : char ty
```

Comment:
A short section title would be good to keep track of what those tests are asserting. Something like
```
(** Known issue: how to print existential types with equality in scope? *)
```